app/index.html

```diff
@@ -57,13 +57,47 @@ <h1 class="brand-logo center">Cartograms 4 All</h1>
     </div>
     <button class="tight waves-effect waves-light btn" id="cust_file">Custom CSV</button>
   -->
- 
+
+    <p style="margin-bottom: 20px; width: 300px">
+    </p>
+
+    <!-- This is my barebones drop down menu now, soon going to add ability to choose country to display-->
+
+
+    <!--div-- class="row">
+      <div class="col s12 m12 push-l3 l8">
+          <div class="card-content">
+            <div class="field-select col s4">
+              <select class="browser-default" id="chooseMap"></select>
+              <label>Country map choice</label>
+            </div>
+
+          </div>
+      </div>
+    </div-->
+
     <div class="row">
       <div class="col s12 m12 push-l2 l8">
         <div class="card card-map">
+
+          <p style="margin-top: 20px; width: 300px">
+          </p>
+
+          <!-- Dropdown Trigger -->
+          <a class='dropdown-button btn' href='#' data-activates='dropdown1'>Choose a Country!</a>
+
+          <!-- Dropdown Structure -->
+          <div class="col l6 s12">
+            <ul id='dropdown1' class='dropdown-content'>
+              <li><a href="#" onclick="chooseCountry(0)">USA(default) </a></li>
```

Comment:
Removed (default), best choice would be to have the currently selected one a different shade, but because of time we can just leave it as is